src/CommunityToolkit.Maui.MediaElement/Extensions/MetaDataExtensions.windows.cs

```diff
@@ -0,0 +1,76 @@
+﻿using Windows.Media;
+using CommunityToolkit.Maui.Core;
+
+namespace CommunityToolkit.Maui.Extensions;
+
+/// <summary>
+/// A class that provides methods to update the system UI for media transport controls to display media metadata.
+/// </summary>
+public class MetaDataExtensions
+{
+	readonly IMediaElement? mediaElement;
+	readonly SystemMediaTransportControls? systemMediaControls;
+	readonly IDispatcher dispatcher;
+	/// <summary>
+	/// Initializes a new instance of the <see cref="MetaDataExtensions"/> class.
+	/// </summary>
+	public MetaDataExtensions(SystemMediaTransportControls systemMediaTransportControls, IMediaElement MediaElement, IDispatcher Dispatcher)
+	{
+		mediaElement = MediaElement;
+		this.dispatcher = Dispatcher;
+		systemMediaControls = systemMediaTransportControls;
+		systemMediaControls.ButtonPressed += SystemMediaControls_ButtonPressed;
```

Comment:
Renaming the method to follow our pattern. Also didn't see the unsubscribe call to this event, can you double check if it will not leak?

```suggestion
		systemMediaControls.ButtonPressed += OnSystemMediaControlsButtonPressed;
```